app/form_models/jobseekers/job_application/employment_history_form.rb

```diff
@@ -1,21 +1,25 @@
 class Jobseekers::JobApplication::EmploymentHistoryForm < Jobseekers::JobApplication::BaseForm
   include ActiveModel::Model
+  include ActionView::Helpers::DateHelper
```

Comment:
this feels like view code in the model layer. It is not possible to add an error which is then rendered later?